### ISSUE TITLE
can u help me

### ISSUE THREAD

**Issue by Viserion-nlper** [sharmi1206/differential-privacy-tensorflow#1]

**Comment by Viserion-nlper** [comment]
Traceback (most recent call last):
  File "D:\Anaconda2\envs\tf1\lib\site-packages\tensorflow_core\python\framework\ops.py", line 1607, in _create_c_op
    c_op = c_api.TF_FinishOperation(op_desc)
tensorflow.python.framework.errors_impl.InvalidArgumentError: Dimension size must be evenly divisible by 250 but is 1 for 'training/Reshape' (op: 'Reshape') with input shapes: [], [2] and with input tensors computed as partial s


how can i sovle it  ,thanks

**Comment by sharmi1206** [comment]
Yes that issue is known to the recent version of TensorFlow-privacy. When I checked in way in Feb, it was running fine, we need to raise this issue with TensorFlow-privacy..

https://hackernoon.com/differential-privacy-with-tensorflow-20-multi-class-text-classification-privacy-yk7a37uh.

here, the code snippets and results are attached..

**Comment by Viserion-nlper** [comment]
Hello, I used the code provided in the webpage, but I still report this error
tensorflow               1.15.0
tensorflow-privacy       0.3.0   


What version did you use at that time? I doubt it was the version problem？

**Comment by Viserion-nlper** [comment]
Hope to get your help
I am seriously thinking about your project
thanks

**Comment by sharmi1206** [comment]
Hello I was using the version available in JAN-FEB 2020. with tensorflow 1.1.15. they made few releases in May for tensorflow-privacy. I also want to resolve this issue for both of us. You already Raised an issue with tensorflow , I could see (https://github.com/tensorflow/privacy/issues/113).. Let me post my comments there. Let us follow up with them

**Comment by sharmi1206** [comment]
> Hope to get your help
> I am seriously thinking about your project
> thanks

Hi,

Please check out my latest code on DP at https://github.com/sharmi1206/Membership_Inference_Attack_DP

It works with latest tensorflow-privacy with TensorFlow version 1.14.

I will revisit the other repository to resolve the previous issue. 

Hope you are able to run this code.
